Symfony/CS/Fixer/PSR2/FunctionCallSpaceFixer.php

```diff
@@ -50,6 +50,13 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
+            // check for ternary operator
+            $endParenthesisIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
+            $nextNonWhiteSpace = $tokens->getNextMeaningfulToken($endParenthesisIndex);
+            if (!empty($nextNonWhiteSpace) and $tokens[$nextNonWhiteSpace]->equals('?')) {
```

Comment:
to give more explanations, ``and`` is not meant as a boolean operator but as a controller flow operator. This is the reason of the totally different precedence of these operators